static/js/stream_list.js

```diff
@@ -437,6 +438,7 @@ export function update_stream_sidebar_for_narrow(filter) {
 
     if (!info.topic_selected) {
         stream_li.addClass("active-filter");
+        pm_list.handle_narrow_deactivated();
```

Comment:
This seems like a very strange place to add this, given what other functions we call nearby -- is this necessary?  It seems most likely that the bug is that we're doing this in the wrong place.

Reply:
Hmm, Thanks for this Tim, fixed this bug too.